mediaplatform_jwp/models.py

```diff
@@ -273,6 +273,15 @@ def get_sources(self):
     #: A property which calls get_sources and caches the result.
     sources = cached_property(get_sources, name='sources')
 
+    def fetch_size(self):
```

Comment:
Can we a) turn this into a ``cached_property`` called size (easy) or b) add a ``size`` field to ``mediaplatform_jwp.models.Video`` which is updated by ``jwpfetch`` (annoying)?